### ISSUE TITLE
Executor tab doesn't show any information when a rebalancing is running

### ISSUE THREAD

**Issue by imans777** [linkedin/cruise-control-ui#128]
When having intra-broker rebalancing, the executor tab shows no information on the progress, although the data is received from cc server, which is shown in the screenshot below on developer tools network tab. It's something about how the UI is trying to fill in the data. In the console tab, there's no error.

CC UI version 0.4.0
CC version 2.5.138

<img width="1534" alt="Screenshot 1403-10-01 at 22 09 29" src="https://github.com/user-attachments/assets/405187a1-7a0a-456a-8412-abf042545fea" />

**Comment by Tasfia-Ara** [comment]
Is this still open? If so I'd like to work on this

**Comment by imans777** [comment]
Yes, and I'd appreciate you work on it 😁 (P.S: I'm not contributor, but we highly use CC/CCFE in our org)
